shanoir-ng-front/src/app/datasets/shared/dataset-type.model.ts

```diff
@@ -31,9 +31,10 @@ export enum DatasetType {
   Statistical = 'Statistical',
   Template = 'Template',
   BIDS = 'BIDS',
+  NIRS = 'NIRS',
```

Comment:
Hi Youenn, I am wondering about this. From the model point of view all is BIDS: MR, NIRS, EEG, iEEG, etc. Why does NIRS stand here next to BIDS?

Reply:
You're right, this update is useless.